docs/api-reference/supervisor-api.md

```diff
@@ -23,89 +23,3075 @@ sidebar_label: Supervisors
   ~ under the License.
   -->
 
-This document describes the API endpoints to manage and monitor supervisors for Apache Druid.
+This document describes the API endpoints to manage and monitor Supervisors for Apache Druid.
 
-## Supervisors
+In this document, `http://<SERVICE_IP>:<SERVICE_PORT>` is a placeholder for the server address of deployment and the service port. For example, on the quickstart configuration, replace `http://<ROUTER_IP>:<ROUTER_PORT>` with `http://localhost:8888`.
 
-`GET /druid/indexer/v1/supervisor`
+## Supervisor information
 
-Returns a list of strings of the currently active supervisor ids.
+### Get an array of active Supervisors
 
-`GET /druid/indexer/v1/supervisor?full`
+#### URL
 
-Returns a list of objects of the currently active supervisors.
+<code className="getAPI">GET</code> `/druid/indexer/v1/supervisor`
+
+Retrieves an array of active Supervisors. Returns a response object that includes an array of strings representing the names of the active Supervisors. If there are no active Supervisors, it returns an empty array.
```

Comment:
```suggestion
Retrieves an array of active supervisors. Returns a response object that includes an array of strings representing the names of the active supervisors. If there are no active supervisors, it returns an empty array.
```

Reply:
Can a supervisor be inactive? If so, is it possible to retrieve an array of inactive supervisors? If the API returns only active supervisors, I think you can drop the word "active".